### DISCUSSION TITLE
Flsun QQ S Pro

### DISCUSSION THREAD

**Discussion by Nemonumpty** [blackfyre/flsun.community#51] [General]
Hi, newbie question.
Can anybody tell me which nozzle type the QQ S Pro uses?
It looks like a Mk8 but the hot-end is V6, only …. V6 nozzles don’t seem to protrude as far.

**Comment by tx0h** [comment]
according to the BOM list of the qq-s (pro), it's a MK8 nozzle but you can print a shorter fan duct when you like to use V6 nozzles.
https://flsun.community/bom/qq-s.html#hotend

this duct is designed for both versions. pick the one you need.

https://www.thingiverse.com/thing:4691289

**Reply by Nemonumpty** [reply]
tx0h many thanks!.
Useful links too 👍
Probably being too cautious 
Hadn’t come across the BOM, just the usual sales blurb.
